.env.homologation.example

```diff
@@ -0,0 +1 @@
+REACT_APP_API_ENDPOINT=http://localhost:8080
```

Comment:
Se vamos usar o env-cmd, a variável de ambiente precisa ser no formato de env do React?

Reply:
Sim, fiz seguindo o padrão do Create React App: [Link aqui](https://create-react-app.dev/docs/deployment/#customizing-environment-variables-for-arbitrary-build-environments)